File: TimeTrackerMt_Api/EWebList.DataRepository/Model/PlanMaster.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TimeTrackerMt.DataRepository.Model
{
	public class PlanMaster
	{
		public int? PlanId { get; set; }

		public string Name { get; set; }

		public int? AmountPerUser { get; set; }

		public bool IsTimeTracking { get; set; }

		public bool IsTaskAndProject { get; set; }

		public int NoOfStorageMonth { get; set; }

		public int NoOfSupportDays { get; set; }
		public int NoOfDepartmentAllowed { get; set; }
		public bool IsAllowClientAccess { get; set; }
		public bool IsDefaultPlan { get; set; }
		public bool IsActive { get; set; }

		public long? CreateBy { get; set; }
	}
}

File: TimeTrackerMt/src/app/_models/Plan-master.ts
export interface PlanMaster {
  PlanId?: number;
  Name: string;
  AmountPerUser: number;
  IsTimeTracking?: boolean;
  IsTaskAndProject?: boolean;
  NoOfStorageMonth: number;
  NoOfSupportDays: number;
  NoOfDepartmentAllowed: boolean;
  IsAllowClientAccess?: boolean;
  IsDefaultPlan?: boolean;
  IsActive?: boolean;
  CreateBy?: boolean;
}

export interface PlanPagination {
  DisplayLength: number;
  DisplayStart: number;
  Search: string;
  SortCol: string;
  SortDir: 'asc' | 'desc';

}
